src/Controller/Backend/Async/UploadController.php

```diff
@@ -55,10 +55,10 @@ public function handleUpload(Request $request): JsonResponse
             return new JsonResponse(['error' => ['message' => 'Invalid CSRF token']], Response::HTTP_FORBIDDEN);
         }
 
-        $area = $request->query->get('area', '');
+        $location = $request->query->get('location', '');
```

Comment:
$locationName

Reply:
It wasn't `areaName` before. I'd like to keep the URL parameters brief.

Reply:
short url parameter is ok, but variable name can be changed